index.html

```diff
@@ -1,5 +1,5 @@
-<input id="'username" type="text" placeholder="username" /><br />
-<input id="'password" type="password" placeholder="password" /><br />
-<button onclick="storeData">SAve Info</button>
+<input id="username" type="text" placeholder="username" /><br />
+<input id="password" type="password" placeholder="password" /><br />
+<button onclick="storeData();">SAve Info</button>
 
```

Comment:
this onClick should work correct?

Reply:
Remove the semicolon. You don't need that.